pkgs/by-name/no/notonoto/package.nix

```
@@ -2,7 +2,10 @@
   lib,
   fetchFromGitHub,
   stdenvNoCC,
+  fontforge,
+  python3Packages,
 }:
+
```

We want to have a python available with the modules, we need to run the included script.

```suggestion

let
  python3' = python3.withPackages (
    ps: with ps; [
      fonttools
      ttfautohint-py
    ]
  );
in
```